Module.php

```diff
@@ -69,16 +70,13 @@ public function disableContentContainer(ContentContainerActiveRecord $container)
      */
     public function getPermissions($contentContainer = null)
     {
-        if ($contentContainer instanceof \humhub\modules\space\models\Space) {
-            return [
-                new permissions\CreatePage(),
-                new permissions\EditPages(),
-                new permissions\AdministerPages(),
-                new permissions\ViewHistory(),
-            ];
-        }
-
-        return [];
+        return [
```

Comment:
Please check whether a contentContainer is set or not.
`if ($contentContainer !== null) { `

Otherwise the permission will also be treated as a global permission:
Which can be specfied on group level (Administration -> Users -> Group -> Permissions)